internal/resource/configmap_test.go

```diff
@@ -343,42 +312,38 @@ prometheus.ssl.cacertfile = /etc/rabbitmq-tls/ca.crt
 					instance.Spec.TLS.CaSecretName = "tls-mutual-secret"
 					instance.Spec.Rabbitmq.AdditionalPlugins = additionalPlugins
 
-					expectedRabbitmqConf := iniString(defaultRabbitmqConf(builder.Instance.Name) + `
-			ssl_options.certfile   = /etc/rabbitmq-tls/tls.crt
-			ssl_options.keyfile    = /etc/rabbitmq-tls/tls.key
-			listeners.ssl.default  = 5671
+					expectedRabbitmqConf := iniString("ssl_options.certfile   = /etc/rabbitmq-tls/tls.crt\n" +
```

Comment:
Similar to the comment above, what's the benefit of using iniString here? I found it easier to read in the previous format 🤔